src/features/pages/deriv-ctrader/copy-trading/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { copy_trading_header } from './styles.module.scss'
+import { why_copy_trade } from './data'
+import Container from 'features/components/atoms/container'
+import Flex from 'features/components/atoms/flex-box'
+import FlexBoxContainer from 'features/components/atoms/flex-box/box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+
+const CopyTrading = () => {
+    return (
+        <Container.Fluid>
+            <FlexBoxContainer
+                direction={'col'}
+                margin_block="20x"
+                align={'start'}
```

Comment:
Do not mix up string attributes with evaluated props,  use `align="start"` instead of `align={'start'}`